identify/extensions.py

```diff
@@ -56,6 +56,7 @@
     'ear': {'binary', 'zip', 'jar'},
     'edn': {'text', 'clojure', 'edn'},
     'ejs': {'text', 'ejs'},
+    'ejson': {'text', 'json'},
```

Comment:
I would also add `ejson` so this can be targetted specifically in addition to `json`

Reply:
Done!

Reply:
@asottile Do I need to do something else or will you take care of merging this? 🙏